.github/workflows/build-binaries.yml

```diff
@@ -46,10 +46,12 @@ jobs:
         run: ./node_modules/@oclif/dev-cli/bin/run pack${{ matrix.pack-command }}
       - name: Create packages
         working-directory: ./packages/cli
+        run: ./node_modules/@oclif/dev-cli/bin/run pack -t="${{ matrix.package-targets }}"
```

Comment:
I think it would be better to do this only in Jenkins actually.

You should see in `Jenkinsfile` a step to `Download release assets`, which then copies the assets in an `artefacts` folder for the dpeloyment pipeline.

We could modify the existing `download-release-assets` script to make it so it also create a `latest` copy.

After all, we only need a copy for S3/CDN, not for github release assets.

Reply:
Initially, I wanted to keep the complexity out of Jenkins but yeah it kinda makes sense because it'd affect the user in the repo. (i.e. much files, so confused, wow)